operator/src/test/java/oracle/kubernetes/operator/helpers/ConfigMapHelperTest.java

```diff
@@ -69,6 +69,7 @@
     "model-diff-v1.py",
     "modelInImage.sh",
     "model-wdt-create-filter.py",
+    "wdt_mii_filter.py",
```

Comment:
consider renaming to start with "model" and to use "-" instead of "_". This follows the pattern of other mii related scripts, and ensures model related scripts sort next to each-other in an `ls`.

Reply:
Seems python doesn't like packages/modules using hyphen/dashes in the name when attempting to import:

ERROR: test_wdt_mii_filter (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_wdt_mii_filter
Traceback (most recent call last):
  File "/usr/lib64/python2.7/unittest/loader.py", line 252, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib64/python2.7/unittest/loader.py", line 230, in _get_module_from_name
    __import__(name)
  File "/scratch/lphan/weblogic-kubernetes-operator/operator/src/test/python/test_wdt_mii_filter.py", line 5
    import model-wdt-mii-filter
                ^
SyntaxError: invalid syntax

What do you think about renaming to 'model_wdt_mii_filter.py'?

Reply:
Sure.